modules/apps/change-tracking/change-tracking-spi/src/main/java/com/liferay/change/tracking/spi/display/BaseCTDisplayRenderer.java

```diff
@@ -102,8 +104,11 @@ displayContext, getResourceBundle(displayContext.getLocale()),
 		boolean userAdmin = permissionChecker.isCompanyAdmin(
 			themeDisplay.getCompanyId());
 
-		boolean showAllData = (Boolean)httpServletRequest.getAttribute(
-			"showAllData");
+		CTConfiguration ctConfiguration =
+			ConfigurationProviderUtil.getCompanyConfiguration(
+				CTConfiguration.class, themeDisplay.getCompanyId());
+
+		boolean showAllData = ctConfiguration.showAllData();
```

Comment:
Avoid unnecessary calls: only get the configuration if the user is a company admin.